### PR TITLE
Add uplink tests

### PR DESCRIPTION
**Pull request by dylanlott**
This commit adds an end to end test of the uplink binary against the storj-sim network.

### CODE REVIEW

File: cmd/storj-sim/network.go [hunk 1]
```diff
@@ -171,6 +171,7 @@ func newNetwork(flags *Flags) (*Processes, error) {
 
 				"--server.extensions.revocation=false",
 				"--server.use-peer-ca-whitelist=false",
+				"--pointer-db.auth.api-key", "abc123",
```
Comment (egonelbre) [inline cmd/storj-sim/network.go]: this is not needed

File: scripts/test-uplink.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+cleanup(){
+    rm -rf "$TMPDIR"
+    echo "cleaned up test successfully"
+}
+
+trap cleanup EXIT
+
+BUCKET=bucket-123
+SRC_DIR=$TMPDIR/source
+DST_DIR=$TMPDIR/dst
```
Comment (egonelbre) [inline scripts/test-uplink.sh]: Let's fix this thing in the test-aws script as well.

File: cmd/storj-sim/network.go (hunk 1 above)
Comment (dylanlott) [reply cmd/storj-sim/network.go]: Removing it 👍

File: scripts/test-uplink.sh (hunk 2 above)
Comment (dylanlott) [reply scripts/test-uplink.sh]: Fixed

Comment (egonelbre) [reply scripts/test-uplink.sh]: did you push?

**Pull request merged**